editor/components/block-list/style.scss

```diff
@@ -136,6 +136,9 @@
 	padding-left: $block-padding;
 	padding-right: $block-padding;
 
+	// Break long spaceless words so they don't overflow the block.
+	overflow-wrap: break-word;
```

Comment:
This is totally correct for Firefox; we had to add this to plenty of places in the Add-ons website.

I forget if Firefox is doing the "correct", but annoying, thing here (it feels like it), but yeah… this comes up a lot 😄